turbinia/lib/utils.py

```diff
@@ -97,6 +97,43 @@ def extract_artifacts(artifact_names, disk_path, output_dir, credentials=[]):
   return _image_export(image_export_cmd, output_dir)
 
 
+def extract_custom_artifacts(artifact_names, artifact_definition_file,
```

Comment:
This seems to be basically the same method as `extract_artifacts` with one extra parameter.  Should we just add that parameter as an optional parameter to the other method to reduce some code duplication?